README.md

```diff
@@ -4,6 +4,8 @@
 
 > "Math magicians" is a website for all fans of mathematics. It is a Single Page App (SPA) that allows users to make simple calculations and read a random math-related quote.
 
+![screenshot](./src/images/scrnsht.jpg)
+
```

Comment:
- [ ] Let's add a live demo link to your project so anyone can access it easily.